src/app/services/resource.service.ts

```diff
@@ -40,7 +40,7 @@ export class ResourceService {
    * @returns {Observable<Study[]>}
    */
   getStudies(): Observable<Study[]> {
-    return this.transmartResourceService.getStudies();
+    return Observable.fromPromise(this.transmartStudiesService.studies);
```

Comment:
I would cache studies in transmart resource service directly

Reply:
also, ideally glowing bear knows one backend-mapper and one backend.resource.service, we should hide backend-specific details from glowing bear as much as possible